pages/tutorials/yaml/phases.rst

```diff
@@ -0,0 +1,294 @@
+element names. The keys can be:
+
+- The name of a section within the current file.
+- The name of an input file and a section in that file, separated by a slash,
+  e.g. ``myfile.yaml/my_elements``. If a relative path is specified, the
+  directory containing the current file is searched first, followed by the
+  Cantera data path.
+- The name ``default`` to reference the standard chemical elements.
+
+Example::
+
+    elements:
+    - default: [C, H, Ar]
+    - isotopes: [O18]
+    - myelements.yaml/uranium: [U235, U238]
+
+The order of the elements specified in the input file determines the order of
+the elements in the phase when it is imported by Cantera.
+
+
+Declaring the Species
+^^^^^^^^^^^^^^^^^^^^^
+
+The species present in the phase may be declared in the optional ``species``
+field.
+
+If a ``species`` field is not present, all of the species from the ``species``
+section in the input file will be added automatically. This is equivalent to
+setting the value of the ``species`` field to the string ``all``.
+
+If all of the species to be added to the phase are defined in the ``species``
+section of the current input file, the ``species`` entry can be a list of
+species names from that section, e.g.::
+
+    species: [H2, O2, H2O]
+
+If species are defined in multiple sections, the ``species`` entry can be a list
+of single-key mappings, where the key of each mapping specifies the source and
+the value is either the string ``all`` or a list of species names. Each key
+can be either the name of a section within the current input file or the
+name of a different file and a section in that file, separated by a slash. If a
+relative path is specified, the directory containing the current file is
+searched first, followed by the Cantera data path. Example::
+
+    species:
+    - species: [O2, N2]
+    - more_species: all
+    - subdir/myfile.yaml/species: [NO2, N2O]
+
+The order of species specified in the input file determines the order of the
+species in the phase when it is imported by Cantera.
+
+Species containing elements that are not declared within the phase may be
+skipped by setting the ``skip-undeclared-elements`` field to ``true``. For
+example, to add all species from the ``species`` section that contain only
+hydrogen or oxygen, the phase definition could contain::
+
+    phases:
+    - name: hydrogen-and-oxygen
+      elements: [H, O]
+      species: all
+      skip-undeclared-elements: true
+
+
+The Kinetics Model
+^^^^^^^^^^^^^^^^^^
+
+The kinetics model to be used, if any, is specified in the ``kinetics`` field.
+Supported model strings are:
+
+    - `gas <https://cantera.org/documentation/dev/doxygen/html/de/dae/classCantera_1_1GasKinetics.html#details>`__
+    - `surface <https://cantera.org/documentation/dev/doxygen/html/d1/d72/classCantera_1_1InterfaceKinetics.html#details>`__
+    - `edge <https://cantera.org/documentation/dev/doxygen/html/d0/df0/classCantera_1_1EdgeKinetics.html#details>`__
```

Comment:
This seems even less likely, but is there a better way to link to Doxygen content? I guess this question would be moot if we moved the Doxygen content into Sphinx using Breathe or something similar.

Reply:
Nope, sorry, see #16